lambda/app/src/routes.ts

```diff
@@ -128,17 +128,16 @@ export const setRoutes = (app: any) => {
     }
   });
 
-  app.delete(`/users/:userId`, async (req, res) => {
+  app.post(`/delete-inactive-idir-users`, async (req, res) => {
```

Comment:
Why get rid of the delete and turn it into a post request?

Reply:
I can send payload to delete so using post, also updated endpoint name to make it more meaningful